rpm/amber-web-authorization.spec

```diff
@@ -58,6 +58,7 @@ Requires: libamberwebauthorization = %{version}-%{release}
 %{_datadir}/translations/amber_web_authorization_eng_en.qm
 %{_libdir}/qt5/qml/Amber/Web/Authorization/libamberwebauthorizationplugin.so
 %{_libdir}/qt5/qml/Amber/Web/Authorization/qmldir
+%{_libdir}/qt5/qml/Amber/Web/Authorization/plugins.qmltypes
```

Comment:
Shouldn't that be part of the devel package?

Reply:
qmldir refers to plugins.qmltypes, so splitting that out is not really possible.

Reply:
typeinfo definition in qmldir is supposed to be used by tools like QtC only, so I doubt splitting it out would cause any issues

Reply:
yes, it would just not be nice and there are other cons as well, like that devel packages are not needed for pure QML development but having plugins.qmltypes there would force devs to build-require them.

Note that similar issues exists from the other side as well: it should be enough to have just plugins.qmltypes available in devel environment. But in reality it contains only the info for C++ types and so the QML files need to be present too. So splitting it off is of no help here either.